### PR TITLE
bunch of ex1 and ex2 proposed changes

### PR DESCRIPTION
**Pull request by eranderson921**
I ran through the new exercises and love them.  Along the way I had some minor changes that I believe will be useful for those learners like myself who are 'less technical'.  "Here be Dragons" was super confusing for me so some I added inline comments to my areas of confusion.   

Sorry for such a large PR, next time I'll break it up into much smaller chunks.

### CODE REVIEW

**Comment by eformat** [comment]
Hi @eranderson921 - thanks a lot for these ! they look pretty good. 🍍🍍🍍
I have made some comments inline.
I think it would be good to merge this PR without the individual comments in the docs ? and raise separate issues for those you think still need some work. That would help us track things a lot easier rather than leaving them in the code.

File: docs/1-the-manual-menace/1-the-basics.md [hunk 1]
```diff
@@ -41,10 +44,11 @@ echo ${CLUSTER_DOMAIN}
 echo ${GIT_SERVER}
 ```
 
-8. Check if you can connect to OpenShift. Run the command below. 
+8. Check if you can connect to OpenShift. Run the command below which should result in a "Login successful" reply. Don't forget to make your replacements for \<USERNAME> and \<PASSWORD>.
```
Comment (kattekrab) [inline docs/1-the-manual-menace/1-the-basics.md:47]: I raised the same issue - and a PR for this.  
This wording is probably better though!

**Comment by eformat** [comment]
@eranderson921 this PR needs some updates as suggested and conflicts resolving - else it is hard to merge it as is. thanks.

**Comment by eformat** [comment]
am closing this for now since had no feedback. please feel free to re-open.

**Pull request closed without merging**